CIA 1017 UI plus 1.1.4 fix release updates (#889)
SoftwareAG/c8y-docs: content/users-guide/sensor-app-bundle/register.md

1.  On a desktop or laptop computer, open a web browser and log in to your {{< product-c8y-iot >}} tenant. From the Cockpit application, click **Connect Smartphone** in the right drawer or in the Welcome widget.

    ![Cockpit application](/images/users-guide/csa/csa-connect-smartphone-right-drawer.png)

2.  Follow the instructions in the wizard to step 3, ensuring that the app is installed on the smartphone.

    ![QR code](/images/users-guide/csa/csa-connect-smartphone-wizard-step3.png)

> 3.  From your smartphone, launch the app and tap **Register** in the top right corner of the screen. 

Reviewer: @BeateRixen Do we use "tap" here in the context of "click"?

1.  On a desktop or laptop computer, open a web browser and log in to your {{< product-c8y-iot >}} tenant. In the Device Management application, select **Devices > Registration** from the navigator.

    ![Manual registration](/images/users-guide/csa/csa-device-registration.png)

> 2.  From your smartphone, launch the app and tap **Register** at the top right of the screen. When the camera opens, tap **REGISTER MANUALLY**.

Reviewer: ```suggestion
2.  From your smartphone, launch the app and tap **Register** at the top right of the screen. When the camera opens, tap **Register manually**.
```

> 3. Back in your web browser, click **Register device** (in the top-right of the page) and then select **General**. Choose a meaningful and unique device ID for your smartphone and a group to assign your device to. Click **Next** and then **Complete**. The server will notice the pending registration and wait for your smartphone to show up.

Reviewer: ```suggestion
3. Back in your web browser, click **Register device** at the top-right of the page and select **General**. Choose a meaningful and unique device ID for your smartphone and a group to assign your device to. Click **Next** and then **Complete**. The server will notice the pending registration and wait for your smartphone to show up.
```

> 4. On your smartphone, fill in your tenant (for example "companytenant01"), select your instance (for example "{{< domain-c8y >}}"), and enter your selected device ID. If your instance is not available in the list, you can enter it manually after clicking **Add other instance**. Click **Register** and your device will contact the server and ask to be accepted.

Reviewer: ```suggestion
4. On your smartphone, fill in your tenant, for example, "companytenant01", select your instance, for example, "{{< domain-c8y >}}", and enter your selected device ID. If your instance is not available in the list, you can enter it manually if you click **Add other instance**. Click **Register** for your device to contact the server and ask to be accepted.
```

    ![Manual registration](/images/users-guide/csa/csa-manual-registration-smartphone.png) 

5.  Back on your web browser, your device's card now shows the options to accept or reject the connection. Accept the connection. Your device will be registered with the server and assigned to the selected group.

6.  Back on your smartphone, tap **Done**. Sensor measurements are sent to the server. They can be viewed in the device's dashboard.

For further information about registering a device on the platform manually, refer to [Device Management > Connecting devices](/users-guide/device-management/#connecting-devices).

If you want to deregister from {{< product-c8y-iot >}}, tap **Deregister** in the top right corner of the screen and approve the confirmation dialog. After that you can connect to the same or any other {{< product-c8y-iot >}} instance or tenant.

{{< c8y-admon-info >}}

> Deregistering a device will automatically remove the device from {{< product-c8y-iot >}}.

Reviewer: ```suggestion
Deregistering a device automatically removes the device from {{< product-c8y-iot >}}.
```
